code/modules/clothing/suits/storage.dm

```diff
@@ -1032,6 +1032,44 @@ obj/item/clothing/suit/sweater/blue
 	icon_state = "white_jacket"
 	body_parts_covered = UPPER_TORSO|ARMS
 
+/obj/item/clothing/suit/storage/suitjacket/skylight_coat
+	name = "skylight overcoat"
+	desc = "A sleek black coat worn by those of now known as Skylight Services, once a group with malicious dealings and business practices, now partially reformed under new management of Lonestar. You feel like this overcoat could easily hide an old school submachine gun."
```

Comment:
```suggestion
	desc = "A sleek black coat worn by those of now known as Skylight Services, once a group with malicious dealings and business practices, now partially reformed under new management of Lonestar. This overcoat seems like it could easily hide an old school submachine gun."
```
Don't use "you feel" statements in an item desc, don't tell someone how they feel.